lib/glob-assets.js

```diff
@@ -1,12 +1,11 @@
 const path = require('path');
-const {basename} = require('path');
 const {isPlainObject, castArray, uniqWith, uniq} = require('lodash');
 const dirGlob = require('dir-glob');
 const globby = require('globby');
 const debug = require('debug')('semantic-release:gitlab');
 
-module.exports = async ({cwd}, assets) =>
-  uniqWith(
+module.exports = async ({pkgRoot, cwd}, assets) => {
+  return uniqWith(
```

Comment:
This can stay an arrow function, no?

Reply:
@fgreinacher I'm confused. It's still an arrow function